visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -798,7 +821,7 @@ private List<RegionIn> extractIgnoreList(CheckOptions options) {
         options.getIgnoreRegions() == null ? Arrays.asList() : options.getIgnoreRegions();
 
     List<VisualRegion> visualRegions =
-        options.getIgnoreRegions() == null ? Arrays.asList() : options.getRegions();
+        options.getRegions() == null ? Arrays.asList() : options.getRegions();
```

Comment:
Isn't it the same change as in this PR? https://github.com/saucelabs/visual-sdks/pull/231

Reply:
Yeah, this is the same, but I kept this here for now as without this, it's simply not working. The source branch is @omacranger's, so unless he rebases with `main`, I have to include this.

As said above:
>This contains the fix commits from other pull requests. As the feat/java-local-dom branch is updated to have these commits, or merged to main, the commits will be removed.